config/config.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import "sync"
+
+const DefaultConfigTemplate = `
+planetmint={{ .Planetmint }}
```

Comment:
all strings need to have quotes, e.g. from
```
planetmint={{ .Planetmint }}
```
to
```
planetmint="{{ .Planetmint }}"
```

Whereas int types don't need quotes. So `service-port={{ .ServicePort }}` stays liek this.